### ISSUE TITLE
First stories

### ISSUE THREAD

**Issue by alejandro5042** [alejandro5042/openstory#8]
Let's discuss what they should be!

I think we should have a short quick one that gets the ball rolling. Also, I can add chapter one of my story.

**Comment by tparadis7** [comment]
Good idea on short stories. I think 3-5 "flash fiction" stories that get across some major themes we want to explore further would be a great start
